docs/api/request.md

```diff
@@ -6,57 +6,73 @@ The `HonoRequest` is an object that can be taken from `c.req` which wraps a [Req
 
 Get the values of path parameters.
 
-```ts
+```ts twoslash
+import { Hono } from 'hono'
+const app = new Hono()
+// ---cut---
 // Captured params
-app.get('/entry/:id', (c) => {
+app.get('/entry/:id', async (c) => {
   const id = c.req.param('id')
-  ...
+  //    ^?
```

Comment:
Is this necessary??

Reply:
@nakasyou 

Can you confirm this?

Reply:
I added it to make the return value easier to understand, but it might be unnecessary.

 I think that `c.req.queries()`, `c.req.parseBody()` and `c.req.header()` are easier to understand, but `c.req.param('id')` is not so. Should I remove this?

Reply:
No. If it's intended, it's okay! Thank you!